src/main/scala/org/constellation/primitives/Transaction.scala

```diff
@@ -199,6 +211,14 @@ sealed trait TransactionValidatorNel {
       .product(validateFee(tx))
       .product(validateDuplicate(tx))
       .map(_ ⇒ tx)
+
+  def validateDummyTransaction(tx: Transaction)(implicit dao: DAO): ValidationResult[Transaction] =
+    validateSourceSignature(tx)
+      .product(validateEmptyDestinationAddress(tx))
+      .product(validateDestinationAddress(tx))
+      .product(validateFee(tx))
+      .product(validateDuplicate(tx))
+      .map(_ ⇒ tx)
```

Comment:
I'd add validation for dummy transaction to be sure it has 0L amount all the time